javascript/src/test/components/ChannelMessageReaction/MessageTableReaction.test.js

```diff
@@ -24,6 +38,369 @@ describe("MessageTableReactions tests", () => {
+            "user_profile": {
+                "real_name": "Test Person"
+            },
+            "message_reactions": [{
+                count: 1,
+                name: "test-name"
+            }]
+        }];
+        const testReaction = "test-name";
+        const { getByText } = render(<MessageTableReaction messages = {testMessages} reaction = {testReaction}/>);
+        const testUser = getByText(/Test Person/);
+        const testText = getByText(/@test-person has joined the channel/);
+        const testCount = getByText(/1/);
+        expect(testUser).toBeInTheDocument();
+        expect(testText).toBeInTheDocument();
+        expect(testCount).toBeInTheDocument();
+    });
+    test("row.message_reactions is formatted (channel mention)", () => {
+        useSWR.mockReturnValue({
+            data: [{
+                id: "U017218J9B3",
+                real_name: "test-person"
+            }]
+        });
+        const testMessages = [{
+            "user": "U017218J9B3",
+            "text": "<@U017218J9B3> has joined the channel",
+            "user_profile": {
+                "real_name": "Test Person"
+            },
+            "message_reactions": [{
+                count: 1,
+                name: "test-name"
+            }]
+        }];
+        const testReaction = "test-name";
+        const { getByText } = render(<MessageTableReaction messages = {testMessages} reaction = {testReaction}/>);
+        const testUser = getByText(/Test Person/);
+        const testText = getByText(/@test-person has joined the channel/);
+        const testCount = getByText(/1/);
+        expect(testUser).toBeInTheDocument();
+        expect(testText).toBeInTheDocument();
+        expect(testCount).toBeInTheDocument();
+    });
+    test("Username not found", () => {
+        useSWR.mockReturnValue({
+            data: []
+        });
+        const testMessages = [{
+            "type": "message",
+            "subtype": "channel_join",
+            "ts": "1594143066.000200",
+            "user": "U017218J9B3",
+            "text": "<@U017218J9B3> has joined the channel",
+            "channel": "section-6pm",
+            "user_profile": {
+                "real_name": "Test Person"
+            },
+            "message_reactions": [{
+                count: 1,
+                name: "test-name"
+            }]
+        }];
+        const testReaction = "test-name";
+        const { getByText } = render(<MessageTableReaction messages = {testMessages} reaction = {testReaction}/>);
+        const nameElement = getByText(/@U017218J9B3/);
+        expect(nameElement).toBeInTheDocument();
+    });
+    test("Unembedded https links are clickable", () => {
+        useSWR.mockReturnValue({
+            data: []
+        });
+        const testMessages = [{
+            "user": "U017218J9B3",
+            "text": "Office hours at <https://ucsb.zoom.us/j/89220034995?pwd=VTlHNXJpTVgrSEs5QUtlMDdqMC9wQT09>",
```

Comment:
LAs have said not to use the class zoom link in tests, probably swap it out for a placeholder instead